_layouts/docs-v2.html

```diff
@@ -0,0 +1,116 @@
+                            </svg>
+                        </button>
+                    </div>
+                </form>
+
+                <div class="svg-icons" style="height: 0; width: 0; position: absolute; visibility: hidden">
+                    <svg xmlns="http://www.w3.org/2000/svg">
+                        <symbol id="sbx-icon-clear-3" viewBox="0 0 40 40"><path d="M16.228 20L1.886 5.657 0 3.772 3.772 0l1.885 1.886L20 16.228 34.343 1.886 36.228 0 40 3.772l-1.886 1.885L23.772 20l14.342 14.343L40 36.228 36.228 40l-1.885-1.886L20 23.772 5.657 38.114 3.772 40 0 36.228l1.886-1.885L16.228 20z" fill-rule="evenodd"/></symbol>
+                        <symbol id="sbx-icon-search-13" viewBox="0 0 40 40"><path d="M26.806 29.012a16.312 16.312 0 0 1-10.427 3.746C7.332 32.758 0 25.425 0 16.378 0 7.334 7.333 0 16.38 0c9.045 0 16.378 7.333 16.378 16.38 0 3.96-1.406 7.593-3.746 10.426L39.547 37.34c.607.608.61 1.59-.004 2.203a1.56 1.56 0 0 1-2.202.004L26.807 29.012zm-10.427.627c7.322 0 13.26-5.938 13.26-13.26 0-7.324-5.938-13.26-13.26-13.26-7.324 0-13.26 5.936-13.26 13.26 0 7.322 5.936 13.26 13.26 13.26z" fill-rule="evenodd"/></symbol>
+                    </svg>
+                </div>
+
+                {% for section in page.docs-v2 %}
+                    {% assign isCurrentSection = false %}
+                    {% for item in section.items %}
+                        {% capture itemUrl %}{{ item[0] | replace: '.md', '.html' }}{% unless item[0] contains ".md" %}/{% endunless %}{% endcapture %}
+                        {% if itemUrl == page.url %}
+                            {% assign isCurrentSection = true %}
+                        {% endif %}
+                    {% endfor %}
+                    <h3 class="toc-title{% if isCurrentSection %} active{% endif %}">{{ section.title }}</h3>
+                    <ul class="toc-links">
+                        {% for item in section.items %}
+                            {% capture itemUrl %}{{ item[0] | replace: '.md', '.html' }}{% unless item[0] contains ".md" %}/{% endunless %}{% endcapture %}
+                            <li><a href="{{ itemUrl }}">{{ item[1] }}</a></li>
+                        {% endfor %}
+                    </ul>
+                {% endfor %}
+            </div>
+
+            <div class="col-md-8 docs-content">
+                <h1 class="page-header">{{ page.title }}</h1>
+                <div class="edit-panel" style="margin: 0 0 1em 0;">
+                    <a href="{{ site.slim_website_repo }}{{page.path}}" target="_blank" class="btn btn-default btn-xs"><i class="fa fa-github"></i> Edit This Page</a>
+                    <select onchange="this.options[this.selectedIndex].value && (window.location = this.options[this.selectedIndex].value);">
+                        <option value="/docs/v2" selected>v2</option>
+                        <option value="/docs/">v3</option>
```

Comment:
I think the value of this should be /docs/v3 rather than just /docs/. I would rather not rely on the redirect for this.

Reply:
Same reasoning here as https://github.com/slimphp/Slim-Website/pull/282#discussion_r167595152